elements/lisk-utils/test/math/q.spec.ts

```diff
@@ -0,0 +1,325 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { Q_OPERATION, q, q96 } from '../../src/math';
+
+/**
+ * For the test cases `binary` represents the calculation result in binary.
+ */
+describe('Q', () => {
+	//  In this test Q8 is used as example because 8 bit is easier to generate the tests.
+	// For Q8 after 9th bit should be ignored
+	const base = 8;
```

Comment:
I think that the tests would be more relevant if the majority of them would use `base = 96`, with only a few tests using another base. Not the other way around.

That would make our tests more realistic, and also provide some sort of stress testing.

Reply:
with base 96, it's quite difficult to check the edge case, where the bits are truncated.
Also, there is tests for q96 as well